### DISCUSSION TITLE
Which tags in cvp are set by cv_deploy and generate_cv_tags

### DISCUSSION THREAD

**Discussion by kmueller68** [aristanetworks/avd#4680] [Q&A]
Hello.

The eos_designs `generate_cv_tags` and also the new `cv_deploy` role will set tags within CVP for devices and interfaces.

`cv_deploy` docs: Deploys device and interface Tags for one or more devices.
Is there anywhere a list, which tags cv_deploy will exactly configure in CVP for a campus fabric and for a dc l3ls fabric?  

`generate_cv_tags` docs: can generate CloudVision Tags that assist CloudVision with rendering the Topology correctly.

Is it just these 5 tags or will it set anymore else for campus and dc fabrics?
```
topology_hint_fabric | The overall fabric that the devices pertains to. Useful for multi-fabric deployments. | fabric_name
topology_hint_datacenter | The datacenter to which the devices belongs. Helpful for multi-dc deployments. | dc_name
topology_hint_pod | The pod to which the devices belongs. | pod_name
topology_hint_rack | The physical rack in which the device is located. | rack defined on node or node_group
```

Thank you very much,
Karlheinz

**Comment by kmueller68** [comment]
I discovered the behaviours now:

1. cv_deploy is not setting any tags in CVP by default based on information available in the yamls.
You have to define your own tags via `generate_cv_tags:` and cv_deploy will push it to cvp.

And `generate_cv_tags:` will set by default only the 4 above listes topology hint tags.; no more.

**Reply by ClausHolbechArista** [reply]
Thank you for investigating yourself. You can add custom tags under `generate_cv_tags` which can source dynamic values from structured config or you can use inline jinja to source dynamic values from other ansible variables.
